python/tk_multi_loader/delegate_publish.py

```diff
@@ -0,0 +1,178 @@
+import sgtk
```

Comment:
I will tag below the methods that have actually been updated from the original code.